pkg/operator/ceph/object/controller.go

```diff
@@ -263,18 +264,23 @@ func (r *ReconcileCephObjectStore) reconcile(request reconcile.Request) (reconci
 				close(r.objectStoreChannels[cephObjectStore.Name].stopChan)
 			}
 
-			response, okToDelete := r.verifyObjectBucketCleanup(cephObjectStore)
-			if !okToDelete {
-				// If the object store cannot be deleted, requeue the request for deletion to see if the conditions
-				// will eventually be satisfied such as the object buckets being removed
-				return response, nil
+			deps, err := CephObjectStoreDependents(r.context, r.clusterInfo, cephObjectStore)
```

Comment:
IMO it is better to add the functionality of `verifyObjectUserCleanup()` to `CephObjectStoreDependents()`. If planning to consider `buckets` for `CephObjectStoreDependents()`, better put this function calling `verifyObjectUserCleanup()` right?

Reply:
That functionality is tracked by a separate issue and will be a separate PR. This is only for blocking when buckets exist. There is currently already code that validates object store users.

Reply:
After your comment, I have been thinking that it might be good just to include that in this PR since the current functionality blocks and doesn't report the CephObjectStoreUser dependencies in the same way. I think it's for the best to report it all the same way.